sidechain/consensus/slots/Cargo.toml

```diff
@@ -23,14 +23,13 @@ sp-consensus-slots = { version = "0.10.0-dev", default-features = false, git = "
 sp-runtime = { version = "4.0.0-dev", default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "master"}
 
 # local deps
-its-primitives = { path = "../../primitives", default-features = false }
-its-consensus-common = { path = "../common", default-features = false }
 itp-sgx-io = { path = "../../../core-primitives/sgx/io", default-features = false }
 itp-time-utils = { path = "../../../core-primitives/time-utils", default-features = false }
 itp-types = { path = "../../../core-primitives/types", default-features = false }
+its-primitives = { path = "../../primitives", default-features = false }
+its-consensus-common = { path = "../common", default-features = false }
 
```

Comment:
Haha, why was this changed? It is still no alphabetically ordered.

Reply:
Ordered it alphabetically now :)